### PR TITLE
Scip v900

### PR DESCRIPTION
**Pull request by ju-manns**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -15,13 +15,13 @@ package:
...
   - url: https://scipopt.org/download/release/scipoptsuite-{{ scip_version }}.tgz
-    sha256: a3c1b45220252865d4cedf41d6327b6023608feb360d463f2e68ec4ac41cda06
+    sha256: aa9c5456810d7425d2ad409364b3a069ed3a492331cc7256aae9b64f2591b66c
     folder: scipoptsuite
     patches:
       - 0002-Require-PCRE-on-WIN32.patch
 
 build:
-  number: 7
+  number: 4
```
Comment (AntoinePrv) [inline recipe/meta.yaml]:
```suggestion
  number: 0
```
We are starting back to zero with this version update (it should be checked at the PR, not commit level).

**Review by AntoinePrv — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (ju-manns) [reply recipe/meta.yaml]: I see. Thanks for reviewing!

**Review by AntoinePrv — APPROVED** [review]

**Comment by ju-manns** [comment]
@AntoinePrv we are ready to release :) can the branch be merged or do we need to update anything because some weeks have passed?

**Comment by AntoinePrv** [comment]
@conda-forge-admin, please rerender

**Comment by AntoinePrv** [comment]
@ju-manns, sorry, I thought you had write permission. Are you interested to join maintainers here?

**Comment by ju-manns** [comment]
> @ju-manns, sorry, I thought you had write permission. Are you interested to join maintainers here?

@AntoinePrv Yeah, that would be great!

**Pull request merged**